### PR TITLE
update deploy document

### PR DESCRIPTION
**Pull request by kissmydb**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: ticdc-deployment-topology.md [hunk 1]
```diff
@@ -32,3 +32,4 @@ aliases: ['/docs-cn/dev/reference/tools/ticdc/deploy/','/docs-cn/dev/ticdc/deplo
 > **注意：**
 >
 > - 无需手动创建配置文件中的 `tidb` 用户，TiUP cluster 组件会在部署主机上自动创建该用户。可以自定义用户，也可以和中控机的用户保持一致。
```
Comment (yikeke) [inline ticdc-deployment-topology.md:34]:
```suggestion
> - 无需手动创建配置文件中的 `tidb` 用户，TiUP cluster 组件会在目标主机上自动创建该用户。可以自定义用户，也可以和中控机的用户保持一致。
```

File: tidb-binlog-deployment-topology.md [hunk 2]
```diff
@@ -39,5 +39,5 @@ category: how-to
 > **注意：**
 >
 > - 编辑配置文件模版时，如无需自定义端口或者目录，仅修改 IP 即可。 
->
 > - 无需手动创建配置文件中的 `tidb` 用户，TiUP cluster 组件会在部署主机上自动创建该用户。可以自定义用户，也可以和中控机的用户保持一致。
```
Comment (yikeke) [inline tidb-binlog-deployment-topology.md:42]:
```suggestion
> - 无需手动创建配置文件中的 `tidb` 用户，TiUP cluster 组件会在目标主机上自动创建该用户。可以自定义用户，也可以和中控机的用户保持一致。
```

File: tiflash-deployment-topology.md [hunk 3]
```diff
@@ -34,4 +34,5 @@ category: how-to
 
 > **注意：**
 >
-> 无需手动创建配置文件中的 `tidb` 用户，TiUP cluster 组件会在部署主机上自动创建该用户。可以自定义用户，也可以和中控机的用户保持一致。
+> - 无需手动创建配置文件中的 `tidb` 用户，TiUP cluster 组件会在部署主机上自动创建该用户。可以自定义用户，也可以和中控机的用户保持一致。
```
Comment (yikeke) [inline tiflash-deployment-topology.md:37]:
```suggestion
> - 无需手动创建配置文件中的 `tidb` 用户，TiUP cluster 组件会在目标主机上自动创建该用户。可以自定义用户，也可以和中控机的用户保持一致。
```

File: geo-distributed-deployment-topology.md [hunk 4]
```diff
@@ -82,4 +82,6 @@ category: how-to
 
 > **注意：**
 >
-> 无需手动创建配置文件中的 `tidb` 用户，TiUP cluster 组件会在部署主机上自动创建该用户。可以自定义用户，也可以和中控机的用户保持一致。
+> - 无需手动创建配置文件中的 `tidb` 用户，TiUP cluster 组件会在部署主机上自动创建该用户。可以自定义用户，也可以和中控机的用户保持一致。
+> - 如果部署目录配置为相对路径，会部署在用户家目录下。
+
```
Comment (yikeke) [inline geo-distributed-deployment-topology.md:87]:
```suggestion
```
To fix lint

File: geo-distributed-deployment-topology.md [hunk 5]
```diff
@@ -82,4 +82,6 @@ category: how-to
 
 > **注意：**
 >
-> 无需手动创建配置文件中的 `tidb` 用户，TiUP cluster 组件会在部署主机上自动创建该用户。可以自定义用户，也可以和中控机的用户保持一致。
+> - 无需手动创建配置文件中的 `tidb` 用户，TiUP cluster 组件会在部署主机上自动创建该用户。可以自定义用户，也可以和中控机的用户保持一致。
```
[the remark this answers is not in the document]
Comment (kissmydb) [reply geo-distributed-deployment-topology.md:85]: ok

File: production-deployment-using-tiup.md [hunk 6]
```diff
@@ -186,4 +186,4 @@ tiup cluster display tidb-test
 mysql -u root -h 10.0.1.4 -P 4000
 ```
 
-更多方式可参考[验证集群运行状态](/post-installation-check.md)。
+对于监控系统、dashboard，以及更多验证方式可参考[验证集群运行状态](/post-installation-check.md)。
```
[the remark this answers is not in the document]
Comment (kissmydb) [reply production-deployment-using-tiup.md:189]: 不是这个含义，我再改一下

File: production-deployment-using-tiup.md [hunk 7]
```diff
@@ -186,4 +186,4 @@ tiup cluster display tidb-test
 mysql -u root -h 10.0.1.4 -P 4000
 ```
 
-更多方式可参考[验证集群运行状态](/post-installation-check.md)。
+此外，也需要验证监控系统、dashboard 的运行状态，以及简单命令的执行，验证方式可参考[验证集群运行状态](/post-installation-check.md)。
```
Comment (yikeke) [inline production-deployment-using-tiup.md:189]:
```suggestion
此外，也需要验证监控系统、TiDB Dashboard 的运行状态，以及简单命令的执行，验证方式可参考[验证集群运行状态](/post-installation-check.md)。
```

File: TOC.md [hunk 8]
```diff
@@ -66,8 +67,9 @@
       + [BR 备份与恢复场景示例](/br/backup-and-restore-use-cases.md)
   + [日常巡检](/daily-inspection.md)
   + [TiCDC 运维操作及任务管理](/ticdc/manage-ticdc.md)
-  + [TiUP 常用运维操作](/maintain-tidb-using-tiup.md)
   + [TiFlash 常用运维操作](/tiflash/maintain-tiflash.md)
+  + [TiUP 常用运维操作](/maintain-tidb-using-tiup.md)
+  + [Ansible 常用运维操作](maintain-tidb-using-ansible.md)
```
Comment (yikeke) [inline TOC.md:72]:
```suggestion
  + [Ansible 常用运维操作](/maintain-tidb-using-ansible.md)
```

**Pull request merged**